parachains/integration-tests/collectives/0_xcm/2_reserve.yml

```diff
@@ -42,43 +40,10 @@ tests:
                 - name: xcmPallet.Attempted
                   chain: *relay_chain
                   attributes:
-                    - type: XcmV2TraitsOutcome
+                    - type: XcmV3TraitsOutcome
                       xcmOutcome: Complete
-                      # the weight must be static
-                      value: *weight_to_send_reserve
                 - name: dmpQueue.ExecutedDownward
                   chain: *collectives_parachain
                   attributes:
-                    - type: XcmV2TraitsOutcome
+                    - type: XcmV3TraitsOutcome
                       xcmOutcome: Incomplete
-                      # the weight must be static
-                      value: [1000000000, UntrustedReserveLocation]
-
-      - name: Reserve assets from Collectives Parachain to Relay Chain fails
```

Comment:
Why is this test no longer needed?

Reply:
here the answer https://github.com/paritytech/cumulus/pull/2221/commits/5f3fda69b0352fcbdcdd3192f14c6a20a9bf1b2d